src/components/form/businessCreateProjectForm.tsx

```diff
@@ -217,42 +189,55 @@ export function CreateProjectBusinessForm() {
     control: form.control,
   });
 
+  // A function to add skills to a specific profile
+  const handleAddSkillToProfile = (profileIndex: any, skill: any) => {
+    const currentSkills = form.getValues(`profiles.${profileIndex}.skills`);
+    if (!currentSkills.includes(skill)) {
+      form.setValue(`profiles.${profileIndex}.skills`, [
+        ...currentSkills,
+        skill,
+      ]);
+    }
+    if (!currSkills.includes(skill)) {
+      setCurrSkills([...currSkills, skill]);
+    }
+  };
+
+  // A function to remove skills from a specific profile
+  const handleRemoveSkillFromProfile = (profileIndex: any, skill: any) => {
+    const updatedSkills = form
+      .getValues(`profiles.${profileIndex}.skills`)
+      .filter((s) => s !== skill);
+    form.setValue(`profiles.${profileIndex}.skills`, updatedSkills);
+
+    setCurrSkills(currSkills.filter((skill) => skill !== skill));
+  };
+
   async function onSubmit(data: ProfileFormValues) {
     setLoading(true);
     try {
-      console.log('Form body:', {
+      await axiosInstance.post(`/project/${user.uid}/project`, {
```

Comment:
Don't do blind post like this, check if your post api was succesful by checking if you get correct status and expected response. Ideally a succesful POST should return 201 as status and the entity(that was created) value in response body